### DISCUSSION TITLE
PAR absorption on the "side" of tree crowns?

### DISCUSSION THREAD

**Discussion by adamhb** [NGEET/fates#1077] [Q&A]
Hi all,

I've been reducing the canopy crown areas of conifer pfts to try to increase density and basal area. When you reduce the canopy crown area the PAR absorption goes down. This is expected to an extent, but some trees absorb a lot of light on the side of their crowns. This made me wonder if FATES allows trees to absorb light on the side of their crowns or if everything comes in the top and then is attenuated? I see that solar angle influences the rate of light attenuation through leaf layers (with lower angles attenuating less rapidly) which perhaps is all you need to capture this? But I was wondering if anyone had other thoughts on this?

Adam

**Comment by rgknox** [comment]
@adamhb , my understanding is that like the two-stream model, the Norman scheme assumes the vegetation is in an infinite and homogenous horizontal plane. So, like you allude, there is no representation of the the geometry of the crowns and how light impacting the side would affect absorption and transmission.  That being said, the schemes do account for the changes in optical depth with the zenith angle changing, but only in how that interacts with the leaf angle distribution and path lengths that assume   a homogenous horizontal plane.  Gordon Bonan once told me to think of the canopy as being transformed into a green slime that  is spread evenly three dimensions.
I suppose the schemes could be adapted in some way to account for crown geometry affects like this, or perhaps there are other schemes that address this issue.  I
SEIB-DGVM does some cool things related to 3D interactions with vegetation: http://seib-dgvm.com/paper.html
And the spartacus model is pretty cool too:  "Fast matrix treatment of 3-D radiative transfer in vegetation
canopies: SPARTACUS-Vegetation 1.1" Robin J. Hogan, Tristan Quaife2, and Renato Braghiere
I'd be keen to learn about others if anyone knows of any.

**Reply by adamhb** [reply]
Thanks @rgknox for your thoughts and for passing on these refs. I'm not familiar with how other VDMs are doing this but the spartacus model approach looks cool. A recent simulation with 3 conifer pfts (with narrow and deep crowns) showed that FATES could achieve site level NPP values within the range of observations. What I'm going to keep an eye on is if the scheme will pose issues for how these narrower crowns compete with the more spreading crowns of an oak.
